include/deal.II/grid/manifold_lib.h

```diff
@@ -283,29 +283,50 @@ class CylindricalManifold : public Manifold<dim,spacedim>
                        const double tolerance = 1e-10);
 
   /**
+   * Compute the cartesian coordinates for a point given in cylindrical
+   * coordinates.
+   */
+  virtual Point<3>
+  pull_back(const Point<spacedim> &space_point) const override;
+
+  /**
+   * Compute the cylindrical coordinates \f$(r, \phi, \lambda)\f$ for the given
```

Comment:
No need for the `\f` here and elsewhere -- we automatically add that in a script, the dollar signs are good enough.